test/topology/test_topology_ip.py

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+"""
+Test consistency of schema changes with topology changes.
+"""
+import time
+from test.pylib.scylla_cluster import ReplaceConfig
+from test.pylib.manager_client import ManagerClient
+from test.topology.util import wait_for_token_ring_and_group0_consistency
+import pytest
+
+
+@pytest.mark.asyncio
+async def test_replace_different_ip(manager: ManagerClient) -> None:
```

Comment:
As a non-expert in this work, I would have appreciated a comment about what scenario this test (and the rest below) are trying to test. What does "replace" refer to specifically and what is "different ip"? Is it testing a server which reboots with its existing data and a different IP address? Or something else?

Reply:
Replace refers to the usual replace operation
https://docs.scylladb.com/stable/operating-scylla/procedures/cluster-management/replace-dead-node.html

The replacing node can use the same IP as the old node or it can use a new IP, the two scenarios are a bit different and have code specific to them in Scylla so they get separate tests.